apps/launch/common-util/constants/stakingContract.ts

```diff
@@ -81,6 +85,7 @@ export const STAKING_TOKEN_ADDRESSES: Addresses = {
   [base.id]: '0x54330d28ca3357F294334BDC454a032e7f353416',
   [arbitrum.id]: '0x3d77596beb0f130a4415df3D2D8232B3d3D31e44',
   [celo.id]: '0xaCFfAe8e57Ec6E394Eb1b41939A8CF7892DbDc51',
+  [mode.id]: '0xcfD1D50ce23C46D3Cf6407487B2F8934e96DC8f9',
```

Comment:
And we all this `olasAddress` :P

Reply:
yeah, I was also confused when updating these constants, renamed :D